src/sql/workbench/common/constants.ts

```diff
@@ -30,7 +30,7 @@ export const InputBoxFocusedKey = new RawContextKey<boolean>('inputBoxFocus', fa
 export const SearchInputBoxFocusedKey = new RawContextKey<boolean>('searchInputBoxFocus', false);
 
 export const enum NotebookLanguage {
-	Notebook = 'notebook',
+	Notebook = 'Notebook',
```

Comment:
On upgrade, will users not see the notebook editor when rehydrating notebook editors? Is this being renamed for consistency?

Reply:
See comment in the top about why this changed - this shouldn't affect anything for rehydrating since for language registry stuff I lower case it all now anyways.